src/components/header/SubHeader.js

```diff
@@ -263,7 +277,7 @@ class SubHeader extends Component {
       handleLetter,
       handlePrint,
       openModal,
-      standardActions,
+      // standardActions,   // Todo: !!! add back this for filtering the standard actions
```

Comment:
comments is not yet in the standard actions . That is the reason I commented this. Will add back once it is present in the BE response.